include/arpa/nameser.h

```diff
@@ -308,15 +309,15 @@ typedef enum __ns_cert_types {
 #define NS_PUT16(s, cp) ns_put16((s), ((cp)+=2)-2)
 #define NS_PUT32(l, cp) ns_put32((l), ((cp)+=4)-4)
 
-unsigned ns_get16(const unsigned char *);
-unsigned long ns_get32(const unsigned char *);
-void ns_put16(unsigned, unsigned char *);
-void ns_put32(unsigned long, unsigned char *);
+unsigned ns_get16(const unsigned char * : count(2));
```

Comment:
Maybe a comment explaining why these are `count(2)` and `count(4)` would be helpful here.